### PR TITLE
CU-86b0yt572

### PR DESCRIPTION
**Pull request by nicod1889**

### CODE REVIEW

**Comment by gabopereyra** [comment]
Task linked: [CU-86b0yt572 Crear entidad Proveedor](https://app.clickup.com/t/86b0yt572)

File: front/src/app/app.config.ts [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import { ApplicationConfig } from '@angular/core';
-import { provideRouter } from '@angular/router';
+import { Supplierouter } from '@angular/router';
```
Comment (santiceron) [inline front/src/app/app.config.ts]: Should be provideRouter

File: front/src/app/app.config.ts [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 
 import { routes } from './app.routes';
 import { MessageService } from 'primeng/api';
 import { provideAnimations } from '@angular/platform-browser/animations';
 
 export const appConfig: ApplicationConfig = {
-  providers: [provideRouter(routes), MessageService, provideAnimations()]
+  Suppliers: [Supplierouter(routes), MessageService, provideAnimations()]
```
Comment (santiceron) [inline front/src/app/app.config.ts]: Should be: "providers: [provideRouter(routes), MessageService, provideAnimations()]"

File: front/package-lock.json [hunk 3]
```diff
@@ -666,7 +666,7 @@
         "semver": "bin/semver.js"
       }
     },
-    "node_modules/@babel/helper-define-polyfill-provider": {
```
Comment (gabopereyra) [inline front/package-lock.json:669]: Wohoooo, error here

**Review by santiceron — CHANGES_REQUESTED** [review]
Review changed lines in front

**Review by santiceron — APPROVED** [review]

**Pull request merged**